src/pages/components/Events.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { EventsList } from "./EventsComps";
+import { Paper, Slide } from '@material-ui/core';
+
+export const Events = () => {
+    const [slidein, setSlideIn] = useState(false);
+
+    return <div align="center" style={{ overflowX: "hidden", overflowY: "hidden" }}><h2>EVENTS</h2>
```

Comment:
Please check if you can avoid inline styling. It makes the code very hard to read, and also wont be easy to change the styling later on.

Reply:
yeah okay